client/components/LinkListPage/Links/Links.jsx

```diff
@@ -20,9 +23,9 @@ const link = (props) => {
 };
 
 link.propTypes = {
-  url: PropTypes.string,
-  title: PropTypes.string,
-  tags: PropTypes.string,
+  url: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  tags: PropTypes.isRequired,
```

Comment:
Tags could perhaps be optional. They might not be required for every resource.

Reply:
Noted!

Reply:
The way I was imagining it, each link will need at least 1 tags (language) and possibly more like level (beginner intermediate advanced)